aiida/work/legacy/job_process.py

```diff
@@ -151,3 +151,9 @@ def _setup_db_record(self):
 
         if parent_calc:
             self._calc.add_link_from(parent_calc, "CALL", LinkType.CALL)
+
+        if self.raw_inputs:
```

Comment:
@sphuber I'm getting ``IntegrityError: null value in column "label" violates not-null constraint`` when running my calculations that go via the ``get_inputs_template`` function to create the input. I think this is because ``_label`` is given in the template, but it's ``None``.
I suggest changing this paragraph to 
```
description = self.raw_inputs.get('_description', None)
if description is not None:
    self._calc.description = description
label = self.raw_inputs.get('_label', None)
if label is not None:
    self._calc.label = label
```
unless there's a better place to fix this issue.